exporter/exporterqueue/queue.go

```diff
@@ -24,7 +25,12 @@ type Queue[T any] queue.Queue[T]
 
 // Settings defines settings for creating a queue.
 type Settings struct {
-	DataType         component.DataType
+
+	// Deprecated: [v0.110.0] Use Signal instead
+	// nolint
```

Comment:
Why do we need nolint here?

Reply:
Even tho I marked this field as deprecated itself, the linter still complained that it was using the deprecated `component.DataType`.